### PR TITLE
Update ABR models

### PR DESCRIPTION
**Pull request by drasmuss**
## Description

Adds two new models, `abr-ai/niagara-9m-batch.en` and `abr-ai/niagara-84m-batch.en`, and updates the revisions of `abr-ai/niagara-19m-batch.en` and `abr-ai/niagara-38m-batch.en`.

Evaluation config for HF Jobs: [abr-ai/open-asr-leaderboard-abr](https://huggingface.co/spaces/abr-ai/open-asr-leaderboard-abr) (a copy of `hf-audio/open-asr-leaderboard-abr` with the Dockerfile moved to torch 2.14)

## Type of change
- [x] New model
- [ ] New dataset
- [ ] Bug fix
- [ ] New feature
- [ ] Other

## New Model Checklist

- [x] If your model is hosted on the Hugging Face Hub, please report your results (WER on each split, average WER, and RTFx) on the HF Hub by adding a `.eval_results/open_asr_leaderboard.yaml` file like [this](https://huggingface.co/CohereLabs/cohere-transcribe-03-2026/blob/main/.eval_results/open_asr_leaderboard.yaml) in the model repo.

| Model | Avg WER | RTFx | AMI-Cleaned | Earnings22-Cleaned-AA-chunked | Gigaspeech-Cleaned | LS Clean | LS Other | SPGISpeech | Monsoon | Voxpopuli-AA-Cleaned |
|---|---|---|---|---|---|---|---|---|---|---|
| niagara-9m-batch.en | 13.49 | 7054 | 20.94 | 21.46 | 14.97 | 6.45 | 14.53 | 5.16 | 17.11 | 7.33 |
| niagara-19m-batch.en | 10.41 | 4324 | 16.17 | 16.10 | 12.72 | 4.74 | 11.02 | 3.67 | 13.15 | 5.73 |
| niagara-38m-batch.en | 8.74 | 4521 | 13.87 | 12.94 | 11.38 | 3.81 | 9.28 | 3.08 | 10.40 | 5.16 |
| niagara-84m-batch.en | 7.46 | 1925 | 12.01 | 10.81 | 10.41 | 3.09 | 7.61 | 2.61 | 8.84 | 4.33 |

### HF Jobs evaluation (open-source models)

- [x] (If a custom configuration is needed) duplicate one of the Space above (click the ⋮ menu → **Duplicate this Space**) to create your own copy, e.g. `your-username/open-asr-leaderboard-mymodel`.
    - [x] Modify the `Dockerfile` to install your model's dependencies, e.g. installing from a specific version/fork of Transformers.
    - [x] Adapt `run_eval.py` for your model — use the [Transformers one](https://huggingface.co/spaces/hf-audio/open-asr-leaderboard-transformers/blob/main/run_eval.py) as a starting point. There is no need to modify/update the normalizer files, as the `run_eval.py` script should save the raw (un-normalized) transcripts and model outptus, and the normalizer from the repo is locally score the model outputs.
    - [x] For models that use `trust_remote_code=True`, please default to a `revision` tag and specify it in your bash script. 
- [x] In this repo, create a folder for your model library and a `submit_jobs.sh` script (use any existing one in this repo as a template) pointing to your Space and a results bucket.

A similar checklist applies for multilingual evaluation (with `run_eval_ml.py` and `submit_jobs_ml.sh`).

####  Key guidelines
- [x] Use the **same decoding hyper-parameters** across all datasets for a given model.
- [x] (If writing a new) `run_eval.py`, it must support **batch processing** and use `normalizer/data_utils.py` for data loading, normalization, and manifest writing.
- [x] Use the **maximum possible batch size** (can differ per dataset) on an H200 GPU.
- [x] Use `torch.compile` and/or relevant optimizations including warmup to maximize RTFx.
- [x] Even if you're not using HF Jobs, prepare an HF space like the [existing models](https://huggingface.co/collections/hf-audio/open-asr-leaderboard-eval-configurations), such that the maintainers can reproduce your results on HF Jobs.
- [x] Please report your results on the relevant public sets, as well as the RTFx.
- [x] Be sure to count the **total number** of parameters. You can get the exact number by adding the following line in your `run_eval.py` script:
```python
print(f"Model size: {sum(p.numel() for p in model.parameters()) / 1e9:.2f}B parameters")
```
- [x] Please provide the following model metadata (see [here](https://huggingface.co/datasets/hf-audio/open-asr-leaderboard-results/blob/main/english_short_latest.csv) for existing models).

License | Size (B) | # Languages | Encoder | Decoder | (Recommended) Link to training data disclosure on model card ([example](https://huggingface.co/nvidia/parakeet-tdt-0.6b-v3#training-dataset))
-- | -- | -- | -- | -- | --
 applied-brain-research-open-license | 0.009 | 1 (en) | State-space model (SSM) | CTC | https://huggingface.co/abr-ai/niagara-9m-batch.en#datasets
 applied-brain-research-open-license | 0.020 | 1 (en) | State-space model (SSM) | CTC | https://huggingface.co/abr-ai/niagara-19m-batch.en#datasets
 applied-brain-research-open-license | 0.038 | 1 (en) | State-space model (SSM) | CTC | https://huggingface.co/abr-ai/niagara-38m-batch.en#datasets
 applied-brain-research-open-license | 0.084 | 1 (en) | State-space model (SSM) | CTC | https://huggingface.co/abr-ai/niagara-84m-batch.en#datasets

- [x] Does the license appear on your model card?

### CODE REVIEW

File: abr/submit_jobs.sh [hunk 1]
```diff
@@ -3,12 +3,12 @@
 # Usage: HF_TOKEN=hf_... bash submit_jobs.sh
 
 # ── Configuration ────────────────────────────────────────────────────────────
-SPACE="${SPACE:-hf-audio/open-asr-leaderboard-abr}"
+SPACE="${SPACE:-abr-ai/open-asr-leaderboard-abr}"
```
Comment (ebezzam) [inline abr/submit_jobs.sh]: I've updated the `hf-audio` version!
```suggestion
SPACE="${SPACE:-hf-audio/open-asr-leaderboard-abr}"
```

**Review by ebezzam — COMMENTED** [review]
@drasmuss thanks for the PR!

I've obtained the results below if you can confirm? Same average WER and slightly different RTFx

## 9m
```
********************************************************************************
Results per dataset:
********************************************************************************
abr/ai-niagara-9m-batch.en | ArtificialAnalysis-Earnings22-Cleaned-AA-chunked__test: WER = 21.46 %, RTFx = 5515.16
abr/ai-niagara-9m-batch.en | VoiceArena-Monsoon_en_IN_test__test: WER = 17.11 %, RTFx = 6949.76
abr/ai-niagara-9m-batch.en | hf-audio-open-asr-leaderboard_ami_cleaned_test: WER = 20.94 %, RTFx = 6674.09
abr/ai-niagara-9m-batch.en | hf-audio-open-asr-leaderboard_gigaspeech_cleaned_test: WER = 14.97 %, RTFx = 7273.18
abr/ai-niagara-9m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.clean: WER = 6.45 %, RTFx = 6304.51
abr/ai-niagara-9m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.other: WER = 14.53 %, RTFx = 6395.60
abr/ai-niagara-9m-batch.en | hf-audio-open-asr-leaderboard_spgispeech_test: WER = 5.16 %, RTFx = 7352.72
abr/ai-niagara-9m-batch.en | hf-audio-open-asr-leaderboard_voxpopuli_cleaned_aa_test: WER = 7.33 %, RTFx = 5682.79

********************************************************************************
Composite Results:
********************************************************************************
abr/ai-niagara-9m-batch.en: WER = 13.49 %
abr/ai-niagara-9m-batch.en: RTFx = 7158.21
********************************************************************************
```

## 19m
```
********************************************************************************
Results per dataset:
********************************************************************************
abr/ai-niagara-19m-batch.en | ArtificialAnalysis-Earnings22-Cleaned-AA-chunked__test: WER = 16.10 %, RTFx = 3476.66
abr/ai-niagara-19m-batch.en | VoiceArena-Monsoon_en_IN_test__test: WER = 13.15 %, RTFx = 4161.64
abr/ai-niagara-19m-batch.en | hf-audio-open-asr-leaderboard_ami_cleaned_test: WER = 16.17 %, RTFx = 4244.51
abr/ai-niagara-19m-batch.en | hf-audio-open-asr-leaderboard_gigaspeech_cleaned_test: WER = 12.72 %, RTFx = 4388.29
abr/ai-niagara-19m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.clean: WER = 4.74 %, RTFx = 3945.62
abr/ai-niagara-19m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.other: WER = 11.02 %, RTFx = 3942.84
abr/ai-niagara-19m-batch.en | hf-audio-open-asr-leaderboard_spgispeech_test: WER = 3.67 %, RTFx = 4445.08
abr/ai-niagara-19m-batch.en | hf-audio-open-asr-leaderboard_voxpopuli_cleaned_aa_test: WER = 5.73 %, RTFx = 3410.54

********************************************************************************
Composite Results:
********************************************************************************
abr/ai-niagara-19m-batch.en: WER = 10.41%
abr/ai-niagara-19m-batch.en: RTFx = 4323.63
********************************************************************************
```

## 38m
```
********************************************************************************
Results per dataset:
********************************************************************************
abr/ai-niagara-38m-batch.en | ArtificialAnalysis-Earnings22-Cleaned-AA-chunked__test: WER = 12.94 %, RTFx = 3569.70
abr/ai-niagara-38m-batch.en | VoiceArena-Monsoon_en_IN_test__test: WER = 10.41 %, RTFx = 4345.38
abr/ai-niagara-38m-batch.en | hf-audio-open-asr-leaderboard_ami_cleaned_test: WER = 13.87 %, RTFx = 4361.30
abr/ai-niagara-38m-batch.en | hf-audio-open-asr-leaderboard_gigaspeech_cleaned_test: WER = 11.38 %, RTFx = 4595.60
abr/ai-niagara-38m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.clean: WER = 3.81 %, RTFx = 4013.01
abr/ai-niagara-38m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.other: WER = 9.28 %, RTFx = 4099.73
abr/ai-niagara-38m-batch.en | hf-audio-open-asr-leaderboard_spgispeech_test: WER = 3.08 %, RTFx = 4673.19
abr/ai-niagara-38m-batch.en | hf-audio-open-asr-leaderboard_voxpopuli_cleaned_aa_test: WER = 5.16 %, RTFx = 3594.91

********************************************************************************
Composite Results:
********************************************************************************
abr/ai-niagara-38m-batch.en: WER = 8.74 %
abr/ai-niagara-38m-batch.en: RTFx = 4528.71
********************************************************************************
```

## 84m
```
********************************************************************************
Results per dataset:
********************************************************************************
abr/ai-niagara-84m-batch.en | ArtificialAnalysis-Earnings22-Cleaned-AA-chunked__test: WER = 10.83 %, RTFx = 1552.22
abr/ai-niagara-84m-batch.en | VoiceArena-Monsoon_en_IN_test__test: WER = 8.82 %, RTFx = 1812.87
abr/ai-niagara-84m-batch.en | hf-audio-open-asr-leaderboard_ami_cleaned_test: WER = 12.01 %, RTFx = 1937.31
abr/ai-niagara-84m-batch.en | hf-audio-open-asr-leaderboard_gigaspeech_cleaned_test: WER = 10.41 %, RTFx = 1972.42
abr/ai-niagara-84m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.clean: WER = 3.09 %, RTFx = 1740.36
abr/ai-niagara-84m-batch.en | hf-audio-open-asr-leaderboard_librispeech_test.other: WER = 7.60 %, RTFx = 1726.72
abr/ai-niagara-84m-batch.en | hf-audio-open-asr-leaderboard_spgispeech_test: WER = 2.61 %, RTFx = 1962.26
abr/ai-niagara-84m-batch.en | hf-audio-open-asr-leaderboard_voxpopuli_cleaned_aa_test: WER = 4.33 %, RTFx = 1506.33

********************************************************************************
Composite Results:
********************************************************************************
abr/ai-niagara-84m-batch.en: WER = 7.46 %
abr/ai-niagara-84m-batch.en: RTFx = 1927.95
********************************************************************************
```

File: abr/submit_jobs.sh (hunk 1 above)
Comment (drasmuss) [reply abr/submit_jobs.sh]: Thanks, I made this change and squashed it into the base commit.

**Comment by drasmuss** [comment]
Results look good to me, thanks for the quick response!

**Comment by ebezzam** [comment]
thanks for following our submission guidelines 🙂 it made it easy to reproduce!

I'll group these results with some others and try to have it published (and this PR merged) by the end of the week!
